.travis.yml

```diff
@@ -20,7 +20,7 @@ script:
 
   - travis_wait 30 bin/dialyzer_check.sh
 
-  - "! git grep ' $' -- \\*.exs | grep -v -e 'exercises/diamond/diamond_test.exs'"
+  - "! git grep ' $' -- \\*.exs"
```

Comment:
`diamond`'s test file doesn't require trailing white space anymore, so exemption is removed